addons/portal/static/tests/tours/skip_to_content.js

```diff
@@ -0,0 +1,24 @@
+/** @odoo-module **/
```

Comment:
I don't think putting this is necessary anymore if the file is is the right folders, to confirm.